pkg/mcp/snapshot/inmemory.go

```diff
@@ -146,6 +146,16 @@ func (s *InMemory) Version(collection string) string {
 	return s.versions[collection]
 }
 
+// Collections is an implementation of Snapshot.Collections
+func (s *InMemory) Collections() []string {
+	result := make([]string, 0, len(s.resources))
+	for col := range s.resources {
+		result = append(result, col)
+	}
+
```

Comment:
nit: inconsistent extra space here vs. the changes in galley/pkg/config/processing/snapshotter/snapshot.go.